internal/config/config.go

```diff
@@ -91,6 +95,7 @@ func (s *Config) Load() error {
 
 	v.Set("config", v.ConfigFileUsed())
 
+	logger.Debug("Viper configs", v.AllSettings())
```

Comment:
do note that this will log, among other things, user password:
```
matias@kiri ~/c/upctl (feat/debug-flag)> 
go run cmd/upctl/main.go server show 00cd5b9f-4996-4374-86f3-bbcb7f650c68 --debug > foo && head foo
2021-04-29T11:15:31.894+0300 DBG | Viper configs  	value:map[client-timeout:1m0s colours:true config:/home/matias/.config/upctl.yaml debug:true output:human password:REMOVED username:devmatiasapi]	@:config@config/config.go:98
```